dotnet/Directory.Packages.props

```diff
@@ -4,39 +4,33 @@
     <!-- https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management -->
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
-
```

Comment:
What's removing all the whitespace? Dotnet format? Resharper?

Reply:
I didn't do anything here explicitly, so it might have been the build tools. I'll see if I can spot the culprit.